extension/content.css

```diff
@@ -698,6 +698,14 @@ a.tabnav-extra[href$="mastering-markdown/"] {
 	display: none !important;
 }
 
+/* For the `add-time-machine-links-to-comments` feature */
+:root .rgh-timestamp-button {
+	padding: 0;
+}
+:root .rgh-timestamp-button .octicon {
+	vertical-align: sub;
```

Comment:
I think this should be `sub` => `middle`. It looks slightly vertically unaligned now.

Reply:
Actually for some reason this isn't being applied at all so it defaults to `middle` (which isn't aligned)

Reply:
`baseline` is the default, not `middle`.

Reply:
When I manually applied `middle` in devtools it looked ok.